ERCS/eip-7281.md

```diff
@@ -0,0 +1,362 @@
+
+In the ideal case, token issuers want the following properties for bridged representations of their tokens:
+
+- Sovereignty. Issuers want to be the logical owners of the canonical representation of their token on L2 and not be locked into any single specific option forever.
+- Fungibility. Token issuers want a single “canonical” representation of their token on each L2, regardless of which bridges (canonical or otherwise) are supported by the issuer.
+- Security. Issuers want to opt into novel secure bridging approaches as they are developed and have granular control over their risk tolerance for any given option.
+- Minimal Liquidity. Issuers want to minimize the costs and complexity of acquiring liquidity for their token on each supported bridge and chain. This property becomes increasingly important as the space eventually expands to 100s or 1000s of connected domains.
+
+There has been some previous work to solve this problem. However, solutions have historically either failed to satisfy all of the above desirable properties or have been custom-built for only a single token ecosystem.
+
+- Celer’s Open Canonical Token Standard proposed a lock-in free standard for bridging tokens to new domains. However, it largely targeted alternative L1s (that don’t already have a canonical bridge) and did not fully solve the fungibility problem. Regardless, Celer’s approach inspired some of the key thinking behind this standard.
+- Maker’s Teleport facility allows for minting and burning canonical DAI between domains. While the approach solves for the desirable properties above, it is highly custom to Maker’s architecture and relies on Maker’s own economic security to function. Circle CCTP similarly solves this problem, but using a mechanism that only centralized token issuers can implement.
+- Token issuer multi-bridge implementations, e.g. Angle protocol, Frax Ferry, and Threshold Network’s tBTC. These examples solve for some or all of the above desiderata and can be applied more broadly to all tokens if coupled with minor additions for compatibility with existing deployed tokens.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Overview
+
+The system proposed below has two main components:
+
+- A standard interface for bridged tokens
+- A wrapper (aka “Lockbox”) that allows existing tokens to adopt the above interface
+
+### Token Interface
+
+All xERC-20 tokens MUST implement the standard ERC-20 interface.  Note that while many of the below functions are inspired by ERC-777, implementers are NOT REQUIRED to adhere to the full ERC-777 specification.
+
+All EIP-XX tokens MUST implement the following interface.
```

Comment:
If you use HTML-style comments, the linter will make sure you fill in placeholders:

```suggestion
All EIP-XX <!-- TODO --> tokens MUST implement the following interface.
```